lua/wikis/commons/Widget/Standings/MatchOverview.lua

```diff
@@ -52,7 +58,7 @@ function MatchOverviewWidget:render()
 				css = {
 					['font-size'] = '0.8em',
 				},
-				children = match.opponents[1].score .. ' - ' .. match.opponents[2].score,
+				children = otherOpponent.score .. ' - ' .. opponent.score,
```

Comment:
Isn't this the wrong order?

Reply:
Guess it makes sense from the context of this specific file, but not in the context of the entire component. I don't really have better names in mind.